### ISSUE TITLE
monkey.py `default` should use bounded `default` method.

### ISSUE THREAD

**Issue by doranid** [brainix/pottery#323]
Currently, monkey.py calls `_default.default(obj)`.
But, `_default.default` is `json.JSONEncoder.default`, which is an unbounded method, so it needs two arguments for a successful call. A possible solution is to define `_default.default = json.JSONEncoder().default` instead.

How to reproduce:
```
 import json
 import pottery
 json.dumps(object())
```
Will throw an error
```
TypeError: default() missing 1 required positional argument: 'o'
```

Instead of an error
```
TypeError: Object of type 'object' is not JSON serializable
```

**Comment by brainix** [comment]
Thanks for the bug report. I've submitted a PR here: https://github.com/brainix/pottery/pull/324

How does it look, @doranid?

**Comment by doranid** [comment]
> Thanks for the bug report. I've submitted a PR here: #324
> 
> How does it look, @doranid?

Seems good, thanks!

**Comment by brainix** [comment]
I've released Pottery 1.1.1 which includes this fix. 🙂
